controlplane/configparser.cpp

```diff
@@ -197,6 +197,32 @@ controlplane::base_t config_parser_t::loadConfig(const std::string& rootFilePath
 	return baseNext;
 }
 
+void config_parser_t::loadConfig_route_bird([[maybe_unused]]controlplane::base_t& baseNext,
+                                            std::vector<controlplane::route::bird_import_t>& birdsImport,
+                                            const nlohmann::json& birdJson)
+{
+	using BirdImport = controlplane::route::bird_import_t;
+	for (const auto& elemJson : birdJson)
+	{
+		BirdImport import;
+
+		if (exist(elemJson, BirdImport::socketStr))
+		{
+			import.socket = elemJson[BirdImport::socketStr];
+		}
+
+		if (exist(elemJson, BirdImport::vrfStr))
+		{
+			import.vrf = elemJson[BirdImport::vrfStr];
+		}
```

Comment:
What if those keys does not exist? We need to use some default value then, right? Otherwise I think it will be two empty strings.
It's better to use `value(const typename object_t::key_type& key, const ValueType& default_value)` method of `nlohmann::json` like so:
```suggestion
		import.socket = elemJson.value(BirdImport::socketStr, "some_default_value");
		import.vrf = elemJson.value(BirdImport::vrfStr, "some_default_value");
```